Makefile

```diff
@@ -4,12 +4,9 @@ TESTDIR=tmp-test-dir-with-unique-name
 PYTEST_COV_ARGS=--cov=$(PROJECT) --cov-config=../pyproject.toml \
 			--cov-report=term-missing --cov-report=xml --cov-report=html \
 			--pyargs ${PYTEST_EXTRA}
-BLACK_FILES=$(PROJECT) doc/conf.py examples
-BLACKDOC_OPTIONS=--line-length 79
-DOCFORMATTER_FILES=$(PROJECT) doc/conf.py examples
-DOCFORMATTER_OPTIONS=--recursive --pre-summary-newline --make-summary-multi-line --wrap-summaries 79 --wrap-descriptions 79
-FLAKEHEAVEN_FILES=$(PROJECT) doc/conf.py examples
+FORMAT_FILES=$(PROJECT) doc/conf.py examples
 LINT_FILES=$(PROJECT) doc/conf.py
+BLACKDOC_OPTIONS=--line-length 79
```

Comment:
Yes, `blackdoc` supports `pyproject.toml`, but I don't think you change works, because `line-length` should be put in the `[tool.black]` section (see https://blackdoc.readthedocs.io/en/latest/options.html). 

So, `line-length` in the `[tool.black]` section will affect both black and blackdoc.

Reply:
The `blackdoc` documentation is wrong/incomplete I think, I tried adding `line-length=70` under `tool.blackdoc` and it works to format **only** the docstring when I run `make format`. The line-length for the regular code lines (controlled by `tool.black` remains at 88.